src/m365/util/commands/accesstoken/accesstoken-get.ts

```diff
@@ -60,10 +60,14 @@ class UtilAccessTokenGetCommand extends Command {
       }
     }
 
-    try { 
+    if (resource.toLowerCase() === 'graph') {
+      resource = auth.defaultResource;
```

Comment:
I would rather not use `defaultResource`. In the unlikely case that we change the default resource, this would break. Let's specify the endpoint specifically.

Also, in this case, I think we should take non-public clouds into account. Meaning we should use the method `Auth.getEndpointForResource` to determine the Graph resource. Am I correct @waldekmastykarz? Context: this PR adds the ability to use `graph` string to fetch a `Graph` access token. Rather than typing the entire resource URL.

Reply:
Done.

Reply:
Correct @milanholemans and a very good catch. We should definitely consider other clouds.